### ISSUE TITLE
Twitter actions - automatically like or retweet? 

### ISSUE THREAD

**Issue by jboesh** [huginn/huginn#2921]
Hi everyone,

I love the Huginn concept and I wonder if i can use it with twitter? to automatically like or retweet based on hashtags? 
what are the Twitter possibilities?

thank you in advance

**Comment by virtadpt** [comment]
That's easy to do.  Use a Twitter Stream Agent to filter for one or more hashtags.  I use a Deduplication Agent to prevent copies due to multiple job workers.  Then use a Twitter Action Agent to retweet any events it gets.

**Comment by jboesh** [comment]
> That's easy to do. Use a Twitter Stream Agent to filter for one or more hashtags. I use a Deduplication Agent to prevent copies due to multiple job workers. Then use a Twitter Action Agent to retweet any events it gets.

Did you manage to get the twittion action agent to work on a particular schedule?  I have set up a controller with a scheduler agent behind it.  but he doesn't seem to listen to It.  it's retweeting every minute now ... when it has to be once every 30 minutes

**Comment by virtadpt** [comment]
Your Twitter Action Agent doesn't say "This type of Agent cannot be scheduled?"  Mine fires whenever it receives an event.

**Comment by jboesh** [comment]
> Your Twitter Action Agent doesn't say "This type of Agent cannot be scheduled?" Mine fires whenever it receives an event.

yes it does, but there are a lot of events that are fired. I don't want my account to be blocked soon. is there a way to schedule the retweet or favorite action? whether to schedule the stream itself?

**Comment by juryrigging** [comment]
I don't know about scheduling per se, but you might want to have a fiddle with the delay agent to slow them down.
